dictionaries/en_US/src/en_US.txt

```diff
@@ -16807,6 +16810,7 @@ republica
 repugnancy
 repulser
 repulsers
+reputational
```

Comment:
Please `reputational` to [`dictionaries/en_shared/src/shared-additional-words.txt`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt).